extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -756,7 +773,7 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 				case "wgDBname":
 					#--- city_dbname
 					$dbw->update(
-						self::table("city_list"),
+						static::table("city_list"),
 						[ "city_dbname" => $value ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "city_dbname" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)